docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+# https://docs.docker.com/compose/compose-file/
+
+version: '3'
+services:
+  node_bb:
```

Comment:
Can you replace all references of `node_bb` to `nodebb` please?

Reply:
Done~